src/lib.rs

```diff
@@ -728,6 +728,16 @@ pub struct ModuleConfig {
     #[serde(default)]
     install: Option<Script>,
 
+    /// Additional environment information to use in the **guest**.
+    #[serde(default)]
+    env: HashMap<String, String>,
+
+    /// Additional resources to copy from the **host** into the **guest**.
+    /// Key: Guest path, relative to the module's directory.
+    /// Value: Guest path, relative to the project directory.
+    #[serde(default)]
+    copy: HashMap<String, String>,
```

Comment:
Is guest the right word? aren't we talking specifically about the Synapse container (or image?)?